src/common/toast/hdsToast.tsx

```diff
@@ -9,26 +9,32 @@ import i18n from '../../locales/i18n';
 import { HDSToastContainerId } from './HDSToastContainer';
 
 interface HDSToastArgs {
+  autoDismiss?: boolean;
   type: DismissableNotificationProps['type'];
   labelText: string;
   text: string;
   toastId?: string;
   translated?: boolean;
 }
 
-interface DismissableNotificationWrapperProps extends HDSToastArgs {
+interface NotificationWrapperProps extends HDSToastArgs {
   toastId: string;
   translated: boolean;
 }
 
-const DismissableNotificationWrapper = ({
-  type,
-  labelText,
-  text,
-  toastId,
-  translated,
-}: DismissableNotificationWrapperProps) => {
+const NotificationWrapper = ({ autoDismiss, type, labelText, text, toastId, translated }: NotificationWrapperProps) => {
   const { t } = useTranslation();
+  const autoDismissTime = 3000;
```

Comment:
For more clarity would it be better to take this outside the scope of the component function, as a constant:
```tsx
const AUTO_DISMISS_TIME = 3000;
```

Reply:
Agree with @Nizar-Rahme we could move this as mentioned to the start of the file.

Reply:
Done.